src/linearmixedmodel.jl

```diff
@@ -226,7 +204,7 @@ diagonal blocks from the conditional variance-covariance matrix,
     s² Λ(Λ'Z'ZΛ + I)⁻¹Λ'
 """
 function condVar(m::LinearMixedModel{T}) where {T}
-    retrms = m.reterms
+    retrms = filter(Base.Fix2(isa, ReMat), m.allterms)
```

Comment:
This change can be reverted.  The rhs of the result is just a (infintesimally) slower version of the line it replaces.

Reply:
I was actually just looking at that line ... I think the original is clearer if there's no real performance difference.